utils/docker/vcluster/README.md

```diff
@@ -0,0 +1,199 @@
+# Using DAOS with Docker
+
+This section describes how to build and deploy Docker images allowing to simulate a small cluster
+using DAOS as backend storage.  This small cluster is composed of the following three nodes:
+
+- The `daos-server` node running a DAOS server daemon managing data storage devices such as SCM or
+  NVMe disks.
+- The `daos-admin` node allowing to manage the DAOS server thanks to `dmg`command.
+- The `daos-client` node using the the DAOS server to store data.
+
+At this time only emulated hardware storage are supported by this Docker platform:
+
+- SCM (i.e. Stoarage Class Memory) are emulated with standard RAM memory.
+- NVMe disks are emulated with a file device.
+
+!!! warning
+    Virtual Docker network such as [bridge](https://docs.docker.com/network/bridge/) are not yet
+    well supported by DAOS.  Thus, one physical network interface of the host should be chosen for
+    being used by the containers through the Docker
+    [host](https://docs.docker.com/network/host/) network.
+
+
+## Prerequisites
+
+To build and deploy the Docker images, `docker` and optionally `docker-compose` shall be available.
+The docker host should have access to the [Docker Hub](https://hub.docker.com/) and
+[CentOS](https://www.centos.org/) official repositories.  Finally,
+[hugepages](https://www.kernel.org/doc/Documentation/vm/hugetlbpage.txt) linux kernel feature shall
+be enabled on the docker host.
+
+The platform was tested and validated with the [CentOS8](https://hub.docker.com/_/centos) official
+docker image.  However other RHEL-like distributions such as
+[rocky8.4](https://hub.docker.com/r/rockylinux/rockylinux) should be supported.
+
+!!! warning
+    Some distributions are not yet well supported such as
+    [rocky8.5](https://hub.docker.com/r/rockylinux/rockylinux): issue with the management of
+    hugepages with the [spdk](https://spdk.io/) library.
+
+
+## Building Docker Images
+
+### Building Base DAOS Image
+
+The first image to create is the `daos-base` image which is not intetended to be used as it, but as
+a base image for building the other three daos images.  This first image could be built directly
+from GitHub with the following command:
+
+```bash
+$ docker build --tag daos-base:centos8 \
+	https://github.com/daos-stack/daos.git#master:utils/docker/vcluster/daos-base/el8
+```
+
+or from a local tree with the following command:
+
+```bash
+$ docker build --tag daos-base:centos8 utils/docker/vcluster/daos-base/el8
```

Comment:
This command should be run from the root of the git checkout, which is itself fine but is inconsistent with the below.

Reply:
```suggestion
$ cd utils/docker/cluster
$ docker build --tag daos-base:centos8 daos-base/el8
```

Reply:
Fixed with commit https://github.com/daos-stack/daos/pull/7556/commits/8ccbf36ca07b92b342baba440f407062e54862fb
Remove all cd command: all command are self contained :)